src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = " ╔╦═╦══╦═╦╗\n" +
+                      " ║╚╗║╔╗║═╣║\n" +
+                      " ║╔╣║╚╝║═╣║\n" +
+                      " ║╚═╩══╩═╝║\n" +
+                      " ╚════════╝\n";
+        System.out.println("Hiiii, my name is\n" + logo );
+        System.out.println("I am here to help :3\n");
+        String line;
+        String[] tasks = new String[100];
```

Comment:
Consider using a constant for maximum amount of tasks. Otherwise readers may be confused why you assigned an array of size 100 in the future. A constant makes it more intuitive that the number did come up randomly.